package/yast2-packager.changes

```diff
@@ -22,6 +22,19 @@ Tue Mar  7 09:37:15 CET 2017 - schubi@suse.de
 - 3.1.122
 
 -------------------------------------------------------------------
+Fri Mar 24 13:53:40 UTC 2017 - mvidner@suse.com
```

Comment:
It is the CASP history, so we have the SP2 history and the CASP history, which one should be the one selected if for example the next maintenance update is for CASP?